### PR TITLE
Implement/database

### PR DESCRIPTION
**Pull request by Busyreadingsomething**
[Trello Ticket](https://trello.com/c/v0Mnwf9N/12-create-database-m)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,3 @@
 node_modules/
-package-lock.json
```
Comment (mlbpro417) [inline .gitignore:2]: good catch here

File: spec/database.test.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+const { Client } = require('pg');
+
+const connectionString = `postgresql://${process.env.POSTGRES_USER}@localhost/${process.env.POSTGRES_DB}` || 'postgres://localhost/thumbnails';
```
Comment (mlbpro417) [inline spec/database.test.js]: really long line here, I'm surprised your linter allowed it

File: spec/database.test.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+describe('Test querying the database', () => {
+  beforeAll(() => client.connect());
+
+  afterAll(() => client.end());
+
+  test('Should get an array length of 300', () => client
+    .query('SELECT * FROM items')
+    .then((res) => {
+      expect(res.rows).toBeInstanceOf(Array);
```
Comment (mlbpro417) [inline spec/database.test.js:20]: does this actually test the length of the array being 300 as described?

File: thumbnails.sql [hunk 4]
```diff
@@ -0,0 +1,2 @@
+DROP DATABASE IF EXISTS puppies;
```
Comment (mlbpro417) [inline thumbnails.sql]: whats the point of these lines?

**Review by mlbpro417 — COMMENTED** [review]
Sent you some things to check out.  Looks pretty good otherwise.

**Comment by Busyreadingsomething** [comment]
@mlbpro417 made the changes. Please check when you get the chance.

**Review by mlbpro417 — APPROVED** [review]
Looks good to me

**Pull request closed without merging**